core/src/main/spark_2.4_3.0_3.1/scala/doric/syntax/NumericColumns2_31.scala

```diff
@@ -46,6 +47,14 @@ private[syntax] trait NumericColumns2_31 {
       (column.elem, numBits.elem)
         .mapN((c, n) => new Column(ShiftRightUnsigned(c.expr, n.expr)))
         .toDC
+
+    /**
+      * Computes bitwise NOT (~) of a number.
+      *
+      * @group Numeric Type
+      * @see [[org.apache.spark.sql.functions.bitwiseNOT]]
+      */
+    def bitwiseNot: DoricColumn[T] = column.elem.map(f.bitwiseNOT).toDC
```

Comment:
Has the expression used underneath changed? Maybe we can go directly to the expression and avoid two interfaces